learn/getting_started/quick_start.md

```diff
@@ -279,6 +279,10 @@ These allow you to configure your Meilisearch instance at launch with [environme
 
 **Instance options affect your entire Meilisearch instance**, not just a single index.
 
+### Configuration file
```

Comment:
The config file only accepts instance options, right? If so, I think it's better to have it as subheading of `### Instance options` instead of a self-sufficient section like `### Index settings`.

```suggestion
#### Configuration file
```

Reply:
It is a subheading of `Instance options` 👀

Reply:
Oh, but aren't both of them on the same heading level?

`### Instance options` on L#276

`### Configuration file` on L#282

Both are level three (`###`), but for configuration file to be a subheading it would have to be level four (`####`), no?

Reply:
I get it now! Yes, you're right 🙏